server/src/test/java/org/elasticsearch/search/aggregations/bucket/missing/MissingAggregatorTests.java

```diff
@@ -427,7 +572,7 @@ private void testCaseWithReduce(Query query,
             try (IndexReader indexReader = DirectoryReader.open(directory)) {
                 final IndexSearcher indexSearcher = newSearcher(indexReader, true, true);
                 final MappedFieldType[] fieldTypesArray = fieldTypes.toArray(new MappedFieldType[0]);
-                final InternalMissing missing;
+                final I missing;
```

Comment:
Maybe call it `result` instead of `missing` because it isn't always `InternalMissing`.

Reply:
Good catch